libs/SalesforceSDK/build.gradle

```diff
@@ -3,15 +3,12 @@ apply plugin: 'com.android.library'
 
 dependencies {
   compile 'com.google.guava:guava:18.0'
-  compile 'org.apache.httpcomponents:httpmime:4.3.2'
-  compile 'org.apache.james:apache-mime4j:0.7.2'
-  compile 'com.android.volley:volley:1.0.0'
+  compile 'com.squareup.okhttp3:okhttp:3.2.0'
```

Comment:
The official ```OkHttp``` distribution on ```jCenter``` says ```2.7.5``` is the latest (published on ```March 11, 2016``` [here](https://bintray.com/bintray/jcenter/com.squareup.okhttp%3Aokhttp/2.7.5/view). Where is this coming from? Should we switch to that?